README.md

```diff
@@ -1,77 +1,373 @@
 
-Next up, learning what the heck the command line is. Dig in! You wont be the same after you learn how command the terminal. Work through this tutorial [here](https://tutorial.djangogirls.org/en/intro_to_command_line/)! Here's a [great tool](https://explainshell.com/) to decode some of those spell like chunks of texts you've been blindly copy pasting into your terminal. 🔮
 
-### Git
+#### [GitHub](https://github.com/)
 
-Git is used to track who changed what on files, as well as many other things. It's going to be one of your top tools, so bookmark the [documentation](https://git-scm.com/doc).  Now, work through [this](https://try.github.io/levels/1/challenges/1) hands on tutorial, don't worry it's really quick. Now take a look at [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow), it may seem a little advanced, but this is how we use git within Vets Who Code. The reason is because that's how the real world works with Git.
+GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere.
 
+You can use GitHub to build software alongside millions of other developers. You can also use it to learn how to code, or find help with programming problems.
 
-[Github Tutorial](https://github.com/jlord/git-it-electron#what-to-install) (don't worry too much about reading all of these. Wet your pallette and bookmark for later)
+GitHub offers an easy way to share your code with friends and classmates, or collaborate on open-source projects.
 
-### Browser
 
-[Download Chrome](https://www.google.com/intl/en_ca/chrome/) (This is our browser we prefer for our dev environment.)
+#### [Dev.to](https://dev.to/)
 
+If you're a developer looking for a career change or an aspiring developer who wants to learn more, Dev.to is the place to start.
 
-<h2 id="html">HTML & CSS</h2>
+Dev.to is an online community where developers from all backgrounds can share their experiences and expertise. It's also the perfect place for anyone interested in learning about software development to connect with like-minded individuals working in the field.
+
+You can find jobs on Dev.to, as well as connect with recruiters and companies seeking employees with your skillset. You can also learn about new technologies, get advice on making the most of your experience and career path, and share tips on success in this industry.
+
+Whether you're looking for a new job or want to meet other developers, Dev.to is an essential part of any aspiring developer's toolkit!
+ 
+At #VetsWhoCode We Believe that long-form writing is one of the best ways to scale yourself, so you don't have to retain information, showcase how you communicate, and a simple way to advocate for yourself to non-technical employers when you're not writing since everyone on here is in tech so the people will be your audience.
```

Comment:
I think that at VWC we see the most success through built networks and a visible online presence. This could take the form of writing, but is not exclusive to writing. I think whether blogs, email lists, Linkedin posts, tik toks, youtube videos, Twitch streaming, meetups, conferences, what matters is being intentional about building your network.